web/modules/custom/joinup_communities/tallinn/tallinn.post_update.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for Tallinn.
+ */
+
+use Drupal\node\Entity\Node;
+
+/**
+ * Install the 'embed_block' module and add the dashboard block in page content.
+ */
+function tallinn_post_update_add_block() {
+  \Drupal::service('module_installer')->install(['embed_block']);
+
+  // The 'Implementation monitoring' page node ID equals 701254.
+  /** @var \Drupal\node\NodeInterface $custom_page */
+  $custom_page = Node::load(701254);
```

Comment:
Can't we use a UUID for this? That is guaranteed to be unique, and then we can also create this custom page on new installs.

Currently when doing a clean install, or when running the Behat tests, this page is not available. This explains why I cannot see the block containing the report in the Tallinn collection.

Reply:
I removed the content creation on module install. This was wrong decision on my side. @idimopoulos was right that we should provide the content in test. The solution now is:
* For automated tests (Behat) the content is provided by tagging with `@tallinn`.
* For manual testing, you should simply enable the `demo_content` module. I added some sample data there that works well for manual testing.